bundles/io/org.eclipse.smarthome.io.rest.core/src/main/java/org/eclipse/smarthome/io/rest/core/internal/thing/ThingResource.java

```diff
@@ -419,8 +419,7 @@ public Response updateConfiguration(@HeaderParam(HttpHeaders.ACCEPT_LANGUAGE) St
                     ex.getValidationMessages());
             return Response.status(Status.BAD_REQUEST).entity(ex.getValidationMessages(locale)).build();
         } catch (Exception ex) {
-            logger.error("Exception during HTTP PUT request for update config at '{}' for thing '{}': {}",
-                    uriInfo.getPath(), thingUID, ex.getMessage());
+            logger.error("Exception during HTTP PUT request for update config at '{}'", uriInfo.getPath(), ex);
```

Comment:
I guess `thingUID` is also part of `uriInfo.getPath()`.

Reply:
Yes - I didn't see the point in duplicating the UID given we were increasing the logging...